src/loggingFunctions.js

```diff
@@ -9,14 +9,14 @@ Object.keys(config._logLevels).forEach(function(logLevel) {
 		logFunction: logLevel === 'trace' ? logUtil.logTraceWithColor : logUtil.logWithColor,
 		level: config._logLevels[logLevel].level,
 		functionName: config._logLevels[logLevel].letter,
-		color: config.colors[logLevel],
+		color: config._types.log === 0 ? config.colors.logs[logLevel] : config.colors.lines[logLevel],

```

Comment:
I don't think you need the ternary operator here...

The top one adds log functions..so it should always be config.colors.log...while the bottom one does the lines...